package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "mfcl",
-  "version": "3.8.2",
+  "version": "3.8.3",
```

Comment:
Since this includes new features, shouldn't this be version 3.9.0?

Reply:
trueee